common/v2/features/SendAssets/components/fields/AssetDropdown.tsx

```diff
@@ -1,10 +1,19 @@
 import React from 'react';
 import { OptionComponentProps } from 'react-select';
+import styled from 'styled-components';
 
 import { translateRaw } from 'translations';
 import { Asset } from 'v2/types';
 import { AssetSummary, Divider, Dropdown } from 'v2/components';
 
+const DropdownContainer = styled('div')`
+  .has-value > .Select-control > .Select-multi-value-wrapper > .Select-input:only-child {
+    transform: translateY(0%);
```

Comment:
Please add a comment whenever for obscur code.
Drilling down 4 levels to apply an empty translation is surprising.